### PR TITLE
Add dynamic year in footer

### PR DESCRIPTION
**Pull request by m1n0s**

### CODE REVIEW

**Comment by firec0der** [comment]
I think that you have to move this code into some another file.

File: index.html [hunk 1]
```diff
@@ -120,7 +120,7 @@
       <li><a href="https://github.com/Kottans" class="vimeo" target="_blank">Github</a></li>
     </ul>
     <div class="legal">
-      <p> Copyright &copy; 2015 Kottans. All rights reserved.</p>
+      <p> Copyright &copy; <span id="current-year">2016</span> Kottans. All rights reserved.</p>
```
Comment (alexk0ch) [inline index.html]: We can do inline-way also: 

`<p> Copyright &copy; <script>document.write((new Date).getFullYear());</script> Kottans. All rights reserved.</p>`

Comment (shvaikalesh) [inline index.html]: @m1n0s Also it will be awesome to replace `<p>` with `<small>`.

Comment (firec0der) [inline index.html]: I prefer to keep all JS code outside of the HTML. Tastes differ.

Comment (shvaikalesh) [inline index.html]: Browsers have to interrupt parsing/DOM creation to execute `document.write`. However, in this case, it would be rather fast.

Comment (shvaikalesh) [inline index.html]: Can we do this in Jekyll?

**Comment by m1n0s** [comment]
I agree with @firec0der about separating HTML and JS to different files.

Thanks, @shvaikalesh for semantic advices.

**Comment by shvaikalesh** [comment]
It is also cool to set `datetime` attr. However, I see no advantage of doing this client-side, only concerns with no-js users. Take a look at [this](http://alanwsmith.com/jekyll-liquid-date-formatting-examples).

**Comment by sudodoki** [comment]
LGTM! 👍

**Comment by shvaikalesh** [comment]
@sudodoki  `datetime` is not valid. I would rather remove it or replace with compliant option. https://html.spec.whatwg.org/multipage/infrastructure.html#valid-date-string

**Comment by sudodoki** [comment]
@shvaikalesh care to send a small PR?

**Comment by shvaikalesh** [comment]
@sudodoki https://github.com/Kottans/kottans.github.io/pull/63

**Pull request merged**